controller/lib/core/sensors.cpp

```diff
@@ -88,7 +108,21 @@ void zero_sensors() {
       get_raw_sensor_zero_reading(PressureSensors::EXHALATION_PIN);
 }
 
-void sensors_init() {
+void sensors_init(float venturiPortDiameter, float venturiChokeDiameter) {
+  if ((venturiPortDiameter > venturiChokeDiameter) &&
+      venturiChokeDiameter > 0.0f) {
+    venturiPortArea = diameterOfCircleToArea(venturiPortDiameter);
+    venturiChokeArea = diameterOfCircleToArea(venturiChokeDiameter);
+  } else {
+    //@TODO should we indicate to caller that defaults were used since
+    // parameters were invalid?
+  }
+  // Update coefficients used to be able to convert differential pressure to
+  // volumetric flow
+  venturiAreaProduct = venturiPortArea * venturiChokeArea;
+  bernoulliAreaDivisor =
+      1.0f / sqrtf(static_cast<float>(pow(venturiPortArea, 2)) -
```

Comment:
Can avoid these static_casts by using `powf` (or just writing it as `x * x`).  In fact I think as written the compiler may not be able to strength-lower this down to powf automatically.

Reply:
Same above.

Reply:
Yeah don't know why that escaped me, programming late does that I guess.